src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,13 @@
 import React from 'react';
+import { PostInfo } from '../PostInfo';
+import { Post } from '../../types/Post';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <ul className="PostList">
+    {posts.map(post => <PostInfo post={post} key={post.id} />)}
```

Comment:
You want your code maximally clear for those who read it)
```suggestion
    {posts.map(post => (
      <PostInfo post={post} key={post.id} />
    ))}
```